.nvmrc

```diff
@@ -1 +1 @@
-18.16.0
+20
```

Comment:
I have a slight preference here for specifying the exact version that gets installed by chef, just for consistency's sake; in this case, I believe that will be `20.5.1`. But that's just a preference, not a requirement

Reply:
My rationale for unpinning the minor is to receive the latest security updates, I can see dozens of missed CVEs between 18.16 and 18.20, for example. Considering we're only using LTS node versions, the feature set within that version should not change much.

Looks like mini racer 0.13+ has Node 22 support but requires us first upgrading Ruby above 3.0.